### PR TITLE
[ADD] Add partner_multi_relation_extended

### PR DESCRIPTION
**Pull request by Ehtaga**

### CODE REVIEW

File: partner_multi_relation_extended/wizards/res_partner_parent_modification.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        for relation_all in self.contact_id.relation_all_ids:
+            if not relation_all.is_automatic and not relation_all.date_end:
+                relation_all.date_end = fields.Date.today()
+
+        # Archive the former contact
+        self.contact_id.active = False
+
+        # Return the new contact form
+        return {
+            'name': 'New Contact',
```
Comment (ghost) [inline partner_multi_relation_extended/wizards/res_partner_parent_modification.py]: @Ehtaga translate 'New Contact'

File: partner_multi_relation_extended/models/res_partner_relation_all.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    )
+
+    @api.model_cr_context
+    def _auto_init(self):
+        """
+        Add new fields to auto_init
+        """
+        self._additional_view_fields.append('strength')
+        self._additional_view_fields.append('note')
+        self._additional_view_fields.append('is_automatic')
```
Comment (ghost) [inline partner_multi_relation_extended/models/res_partner_relation_all.py:39]: @Ehtaga what is this for ?

File: partner_multi_relation_extended/models/res_partner_relation_strength.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    _description = 'Partner Relation Strength'
+
+    name = fields.Char(
+        'Name',
+        required=True,
+    )
+
+    sequence = fields.Integer(
+        'Sequence',
+        index=True
```
Comment (ghost) [inline partner_multi_relation_extended/models/res_partner_relation_strength.py]: @Ehtaga indexing the column does not help here.

**Comment by ghost** [comment]
@Ehtaga few minor comments, otherwise, LGTM

File: partner_multi_relation_extended/models/res_partner_relation_all.py (hunk 2 above)
Comment (Ehtaga) [reply partner_multi_relation_extended/models/res_partner_relation_all.py:39]: In the original partner_multi_relation_module:
``` 
_additional_view_fields = []
"""append to this list if you added fields to res_partner_relation that
you need in this model and related fields are not adequate (ie for sorting)
You must use the same name as in res_partner_relation.
Don't overwrite this list in your declaration but append in _auto_init:

@api.model_cr_context
def _auto_init(self):
    self._additional_view_fields.append('my_field')
    return super(ResPartnerRelationAll, self)._auto_init()

my_field = fields...
"""```

Comment (ghost) [reply partner_multi_relation_extended/models/res_partner_relation_all.py:39]: @Ehtaga ok thanks

Comment (ghost) [reply partner_multi_relation_extended/models/res_partner_relation_all.py:39]: @Ehtaga ok thanks

File: partner_multi_relation_extended/README.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Atlas Medic
```
Comment (ghost) [inline partner_multi_relation_extended/README.rst]: @Ehtaga oups

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**